### ISSUE TITLE
error in LOLdataset 

### ISSUE THREAD

**Issue by yangcheng001** [weichen582/RetinexNet#9]
there are several mistakes on the correspondance in "our485", like the 79.png, 735.png.

**Comment by JiaqianLi** [comment]
Yes,I also find the problem. The order of low and high images in "our485" do not match, like the 79.png and 111.png. How to solve the problem.

**Comment by SYL-322** [comment]
I found this problem too!

**Comment by Mofafa** [comment]
And the eval images is very similar with the trainset. I dont think this dataset is valuable

**Comment by changruowang** [comment]
I found too

**Comment by Aprilsongsong** [comment]
You're not alone
